crates/sui-bridge/src/types.rs

```diff
@@ -313,6 +316,14 @@ impl Message for BridgeAction {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct EthLog {
+    pub block_number: u64,
+    pub tx_hash: H256,
+    pub log_index_in_tx: u16,
+    pub log: Log,
```

Comment:
Do we still need Log here or could we just keep the essentials?

Reply:
we can pull the essentials out and flat it. let me add a TODO here to clean it up (saving me some efforts for rebasing)